biome-behaviour.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Getting Started
+category: reference
+description: Learn how to set up a new project with Biome.
+---
```

Comment:
This isn't correct, it should not be copied. I just showed you **an example**, but here you have:

```
---
title: How Biome works
description: Understand how the Biome toolchain works
---
```